### PR TITLE
UI for admin's user edit, create.

### LINKED ISSUE #197: Define Edit User page behavior.
The design of edit user page complete. There are no definitions for admin and moderator users.

### PR DESCRIPTION
**Pull request by fisx**
Fixes #197.

(Still needs design, but I'll add the resp. links to the pad.)

### CODE REVIEW

File: src/Frontend/Page/Admin.hs [hunk 1]
```diff
@@ -373,31 +373,41 @@ instance ToHtml AdminViewClasses where
...
 
 chooseRole :: Maybe Role -> Monad m => DF.Form (Html ()) m RoleSelection
 chooseRole mr = DF.choice roleSelectionChoices (selectRole =<< mr)
   where
     selectRole = \case
-        Student _    -> Just RoleStudent
-        ClassGuest _ -> Just RoleGuest
+        Student _    -> Just RoleSelStudent
+        ClassGuest _ -> Just RoleSelClassGuest
         _            -> Nothing  -- FIXME: see RoleSelection
```
Comment (np) [inline src/Frontend/Page/Admin.hs]: Was leaving this unfixed intentional? I would suggest to use the `roleSelection` getter here.

Comment (fisx) [inline src/Frontend/Page/Admin.hs]: Not intentional, thanks!  Will fix.

File: src/Frontend/Page/Admin.hs [hunk 2]
```diff
@@ -403,12 +403,15 @@ roleSelection = to $ \case
     Admin        -> RoleSelAdmin
 
 chooseRole :: Maybe Role -> Monad m => DF.Form (Html ()) m RoleSelection
-chooseRole mr = DF.choice roleSelectionChoices (selectRole =<< mr)
+chooseRole mr = DF.choice roleSelectionChoices (Just . selectRole =<< mr)
```
Comment (np) [inline src/Frontend/Page/Admin.hs]: `selectRole <$> mr`

Comment (np) [inline src/Frontend/Page/Admin.hs]: Actually no `mr ^? _Just . roleSelection`

**Comment by np** [comment]
LGTM please merge when green.

**Comment by andorp** [comment]
LGTM

**Pull request merged**